src/main/kotlin/org/jitsi/nlj/RtpReceiverImpl.kt

```diff
@@ -190,11 +193,12 @@ class RtpReceiverImpl @JvmOverloads constructor(
                                 predicate = PacketPredicate { it is VideoRtpPacket }
                                 path = pipeline {
                                     node(RtxHandler(streamInformationStore, logger))
-                                    node(PaddingTermination())
+                                    node(DuplicateTermination(logger))
+                                    node(RetransmissionRequesterNode(rtcpSender, backgroundExecutor, logger))
```

Comment:
Would it have always been more correct to have `RetransmissionRequester` before video parsing?

Reply:
I think their ordering with respect to each other doesn't matter much; but we want `RestransmissionRequester` before the probing discarder, and `VideoParser` after it.